tools/kmt/kmt.c

```diff
@@ -247,11 +247,11 @@ bool           latency = false;
 bool           recverify = true;
 bool           keybinary = false;
 char          *fieldname_fmt = "field%u";
-uint           fieldnamew_min; /* minimum fieldname width */
-uint           fieldnamew_max; /* maximum fieldname width */
+size_t         fieldnamew_min; /* minimum fieldname width */
+size_t         fieldnamew_max; /* maximum fieldname width */
 uint           fieldcount_max = 2048;
 uint           fieldcount = 1;
-uint           fieldlength = 0;
+size_t         fieldlength = 0;
```

Comment:
I think these should all be signed int to avoid breaking tests of them against the int returned by snprintf()...